jetson/odrive_bridge/src/__main__.py

```diff
@@ -54,7 +54,7 @@ def main():
                 usb_lock.release()
 
             usb_lock.acquire()
-            odrive_bridge.on_event(Event.DISCONNECTED_ODRIVE)
+            odrive_bridge.bridge_on_event(Odrive_Event.DISCONNECTED_ODRIVE)

```

Comment:
can we please not use a mix of camel and snake case? either OdriveEvent , odriveEvent, or odrive_event

Reply:
ya sure